.eslintignore

```diff
@@ -0,0 +1,41 @@
+.env
+
+# OS
```

Comment:
Do we need to specify all this for ESLint? Is it really going to try and lint `DS_Store` files? This looks like a `.gitignore`

Reply:
By default eslint is using gitignore, but as we have lib which is versioned but not code, it should be added in eslint ignore list